core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -90,29 +93,34 @@
   @GeneratedValue(strategy = GenerationType.AUTO)
   private long projectId = AbstractProject.UNSAVED_ID;
 
-  @Transient
+  @OneToMany(targetEntity = SampleImpl.class, fetch = FetchType.LAZY, mappedBy = "project")
   private Collection<Sample> samples = new HashSet<>();
-  @Transient
-  private Collection<Run> runs = new HashSet<>();
-  @Transient
+  @ManyToMany(targetEntity = StudyImpl.class, fetch = FetchType.LAZY)
+  @JoinTable(name = "Project_Study", joinColumns = { @JoinColumn(name = "studies_studyId") }, inverseJoinColumns = {
+      @JoinColumn(name = "Project_projectId") })
   private Collection<Study> studies = new HashSet<>();
-  @Transient
+  @OneToMany(targetEntity = ProjectOverview.class, mappedBy = "project")
   private Collection<ProjectOverview> overviews = new HashSet<>();
-  @Transient
+  @ElementCollection
+  @JoinTable(name = "Project_Issues", joinColumns = { @JoinColumn(name = "project_projectId") })
+  @Column(name = "issueKey")
   private Collection<String> issueKeys = new HashSet<>();
 
-  @Column
-  @Type(type = "progressTypeUserType")
+  @Enumerated(EnumType.STRING)
   private ProgressType progress;
 
-  @OneToOne(targetEntity = ReferenceGenomeImpl.class)
+  @ManyToOne(targetEntity = ReferenceGenomeImpl.class)
   @JoinColumn(name = "referenceGenomeId", referencedColumnName = "referenceGenomeId", nullable = false)
   private ReferenceGenome referenceGenome;
 
-  @Transient
+  @ManyToOne
```

Comment:
These annotations are not constraints or validation. They're used to tell Hibernate how to build efficient queries. Which annotation is used (OneToOne, ManyToOne, etc.) doesn't have to reflect reality or intention. It should match the DB schema. In the schema design, this particular relationship is ManyToOne